compiler/passes/convert-uast.cpp

```diff
@@ -3727,7 +3712,6 @@ struct Converter {
 
       // If there is a type query, extract it from the domain.
       if (lastTypeQuery) {
-        CHPL_ASSERT(isFormalType);
```

Comment:
Could/should this be replaced by `CHPL_ASSERT(inTypeExpression)` ?

Reply:
I think that assert was just checking that we only convert type queries in formal types. Since we already check for that in post-parse-checks I'm not too worried.